plugins/speculation-rules/hooks.php

```diff
@@ -27,7 +27,6 @@ function plsr_print_speculation_rules() {
 	// This workaround is needed for WP 6.4. See <https://core.trac.wordpress.org/ticket/60320>.
 	$needs_html5_workaround = (
 		! current_theme_supports( 'html5', 'script' ) &&
-		version_compare( strtok( get_bloginfo( 'version' ), '-' ), '6.4', '>=' ) &&
```

Comment:
I think this makes sense to still keep since it was limited to 6.4. Otherwise this logic would seem to say that the issue was fixed in 6.5 but it existed for all previous versions. When we bump the minimum version to 6.5 then we can remove the entire condition.

Reply:
Got it. Condition reverted in https://github.com/WordPress/performance/pull/1062/commits/81a78fd5024ece2e0a6ce142a934a2669051713b